src/playwright/github/jobs/node-job-options.ts

```diff
@@ -0,0 +1,15 @@
+// import type {PullRequestFromPatchOptions} from 'projen/lib/github'
```

Comment:
Isn't it just a copy from the `common/github/jobs` folder? Let's remove the duplicate.

Reply:
Fixed